resource-managers/kubernetes/integration-tests/dev/dev-run-integration-tests.sh

```diff
@@ -150,7 +150,10 @@ fi
 
 if [ -n "$DOCKER_FILE" ];
 then
-  properties=( ${properties[@]} -Dspark.kubernetes.test.dockerFile=$(realpath $DOCKER_FILE) )
+  properties=(
+    ${properties[@]}
+    -Dspark.kubernetes.test.dockerFile=$(
+      python3 -c "import os.path; print(os.path.realpath(\"$DOCKER_FILE\"))") )
```

Comment:
@sarutak We should use `realpath` consistently in our Spark bash code.

It seems that the root cause is that 
1) You can a relative path via realpath in some directory
2) And, we moved to another directory.

Reply:
I didn't notice `realpath` in `util.sh` and I used `realpath` command which is installed on Linux.
OK, I'll use our own `realpath` here.